src/components/calculator/PrevalenceControls.tsx

```diff
@@ -194,59 +199,79 @@ export const PrevalenceControls: React.FunctionComponent<{
 
   const isManualEntryCurrently = isManualEntry(data.topLocation)
 
+  const topLocationOptions = [
+    {
+      label: t('calculator.select_location_enter_manually'),
+      value: TOP_LOCATION_MANUAL_ENTRY,
+    },
+    ...Object.keys(locationGroups).flatMap((groupName) => {
+      return locationGroups[groupName].map((locKey) => {
+        return { label: Locations[locKey].label, value: locKey }
+      })
+    }),
+  ]
+  const selectedTopLocation = topLocationOptions.find(
+    (option) => option.value === data.topLocation,
+  )
+
+  const subLocationOptions = !showSubLocation
+    ? []
+    : Locations[data.topLocation].subdivisions.map((locKey) => {
+        return { label: Locations[locKey].label, value: locKey }
+      })
+  const selectedSubLocation = subLocationOptions.find(
+    (option) => option.value === data.subLocation,
+  )
+
   return (
     <React.Fragment>
       <header id="location">
         <Trans>calculator.location_selector_header</Trans>
```

Comment:
this broke

![image](https://user-images.githubusercontent.com/32290/104785545-9b00a900-573f-11eb-9a21-94e0c3150138.png)

Reply:
Fixed! Thanks for the catch!